projects/ngx-text-diff/src/public_api.ts

```diff
@@ -4,4 +4,8 @@
 
 export * from './lib/ngx-text-diff.service';
```

Comment:
I did not look at all the packaged components - but do we need to export every file as a public file, or just the ones we want the user to able to use?